schwab_converter.py

```diff
@@ -4,6 +4,14 @@
 from converter import Converter
 
 
+def _create_mapping(list1, list2):
```

Comment:
Move this function as `_creater_column_mapping` inside the class and annotate it with `@staticmethod`. Also, please use add type hints for the arguments for easier IDE parsing